src/components/layout/MobileNav.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import Link from "next/link";
+import clsx from "clsx";
+
+import { menus } from "./helper";
+
+export interface MobileNavProps {
+  pathname: string;
+  isMobileMenuOpen: boolean;
+  toggleMobileMenu: () => void;
+}
```

Comment:
As mentioned above, interfaces specific to component name, should be used within component only. Better to move to seperate file and rename if you think it can be reused or inherit.